Framework/Reflectometry/src/ReflectometryReductionOneAuto3.cpp

```diff
@@ -587,17 +586,22 @@ void ReflectometryReductionOneAuto3::populateAlgorithmicCorrectionProperties(con
           throw std::runtime_error("Could not find parameter 'polystring' in "
                                    "parameter file. Cannot apply polynomial "
                                    "correction.");
+        setProperty("Polynomial", polyVec[0]);
```

Comment:
Sorry if I'm missing something obvious with this, but I was wondering what's the reason for setting the value on the `ReflectometryReductionOneAuto` property? I notice the same thing is being done on lines 602 and 603 as well.

Reply:
I though it could be easier for tracking history (since you can set those parameters on the 'auto' algorithm). Also to support the unit tests. I have change it to made it like you suggest.